commons/src/main/java/org/calrissian/accumulorecipes/commons/support/qfd/QfdHelper.java

```diff
@@ -141,17 +138,10 @@ public QfdHelper(Connector connector, String indexTable, String shardTable, Stor
         }
 
         if (!connector.tableOperations().exists(this.shardTable)) {
-            connector.tableOperations().create(this.shardTable, false, LOGICAL);
+            connector.tableOperations().create(this.shardTable, true);
```

Comment:
Yes, I did. The logical timestamps were only useful to guarantee that keys didn't squashed. Now that all the keys are always being squashed, the logical timestamp is not necessary. In fact, it will actually stand in the way when we integrate fluo because the timestamp of the physical key is going to need to match the timestamp of the last transaction that occurred to determine which index keys to use.